src/test/java/org/fundacionjala/coding/ariel/Kata1Test.java

```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.coding.ariel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  Kata1 Tests.
+ */
+public class Kata1Test {
+    /**
+     * It checks all the conditions.
+     */
+    @Test
+    public void testSpinWordsIfMoreThatOneWordIsPresent() {
+        Kata1 words = new Kata1();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)